examples/gno.land/r/demo/upgradable/admin/entry.gno

```diff
@@ -0,0 +1,15 @@
+package admin 
```

Comment:
We should consolidate all the upgrade demos in one location, and use distinctive names. If you'd like, I can move my other demos to a different folder. Ideally, we should also maintain a shared README that outlines the differences in each implementation; I believe I have already created one.

Reply:
@moul @mconcat How about we move this PR to https://github.com/gnolang/gno/tree/master/examples/gno.land/r/x/manfred_upgrade_patterns for now as `upgrade_g`, and share the same README until we reach an agreement on the upgradable pattern for mainnet? Once finalized, we can move the final one to https://github.com/gnolang/gno/tree/master/examples/gno.land/r/demo.